src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/FSharp.ProjectItemsSchema.xaml

```diff
@@ -12,5 +12,8 @@
     <ItemType Name="Compile" DisplayName="F# compiler"/>
     
     <FileExtension Name=".fs" ContentType="FSharpFile"/>
+    <FileExtension Name=".fsi" ContentType="FSharpFile"/>
```

Comment:
Is the right default action to include these as part of the compile step?

Reply:
Yes, most definitely.